engines/e_afalg.c

```diff
@@ -362,6 +362,18 @@ static int afalg_fin_cipher_aio(afalg_aio *aio, int sfd, unsigned char *buf,
                         ALG_WARN
                             ("%s(%d): Crypto Operation failed with code %lld\n",
                              __FILE__, __LINE__, events[0].res);
+                        char strbuf[32];
```

Comment:
The declaration must be on beginning of the else block and separated with an empty line from the rest of the block.